kfdefs/base/monitoring/blackbox-exporter/probe.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: Probe
+metadata:
+  name: opf-prober
+spec:
+  jobName: opf-status
+  interval: 60s
+  module: http_2xx
+  prober:
+    url: blackbox-prober.opf-monitoring.svc.cluster.local:80
+    scheme: http
+    path: /probe
+  targets:
+    staticConfig:
+      static:
+      - https://jupyterhub-opf-jupyterhub.apps.zero.massopen.cloud/
```

Comment:
How does a forced http probe to the https endpoint end up behaving?

Reply:
it seems to scrape the https endpoint fine, the blackbox exporter shows the result as `success` for the endpoint. Is there anything else we should be checking to validate it?

Reply:
Hmm, if Prometheus says it's up, it's up. Not much more we could do here.